libsolidity/grammar.txt

```diff
@@ -79,10 +79,12 @@ Expression =
 
 PrimaryExpression = Identifier | BooleanLiteral | NumberLiteral | StringLiteral
 
-FunctionCall = Identifier '(' Expression? ( ',' Expression )* ')'
+FunctionCall = ( MemberAccessFunctionCall | IndexAccessFunctionCall ) '(' Expression? ( ',' Expression )* ')'
```

Comment:
Done - may I also change the `NumberLiteral`

from

`NumberLiteral = '0x'? [0-9]+ (' ' NumberUnit)?`

to

`NumberLiteral = '0x'? [0-9]+ NumberUnit?`

???